.ci/build-mac.sh

```diff
@@ -32,15 +28,39 @@ git checkout f52efee0f18668c6d6de2dec0234b8c4bc54c597
 cd ..
 
 export Qt5_DIR="$WORKDIR/qt-downloader/5.15.2/clang_64/lib/cmake/Qt5"
+
+
 export SDL2_DIR="$BREW_X64_PATH/opt/sdl2/lib/cmake/SDL2"
 
 export PATH="$BREW_X64_PATH/opt/llvm@16/bin:$WORKDIR/qt-downloader/5.15.2/clang_64/bin:$BREW_BIN:$BREW_SBIN:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Library/Apple/usr/bin:$PATH"
 export LDFLAGS="-L$BREW_X64_PATH/lib -Wl,-rpath,$BREW_X64_PATH/lib"
 export CPPFLAGS="-I$BREW_X64_PATH/include -msse -msse2 -mcx16 -no-pie"
 export LIBRARY_PATH="$BREW_X64_PATH/lib"
 export LD_LIBRARY_PATH="$BREW_X64_PATH/lib"
-
-git submodule update --init --recursive --depth 1
+export LLVM_DIR
+LLVM_DIR="BREW_X64_PATH/opt/llvm@16"
+git submodule update --init ./3rdparty/asmjit && \
+git submodule update --init ./3rdparty/cubeb && \
+git submodule update --init ./3rdparty/curl && \
+git submodule update --init ./3rdparty/ffmpeg && \
+git submodule update --init ./3rdparty/flatbuffers && \
+git submodule update --init ./3rdparty/glslang && \
+git submodule update --init ./3rdparty/GPUOpen && \
+git submodule update --init ./3rdparty/hidapi && \
+git submodule update --init ./3rdparty/libpng  && \
+git submodule update --init ./3rdparty/libusb && \
+git submodule update --init ./3rdparty/miniupnp && \
+# TODO add a prebuilt library to moltenvk like ffmpeg to speed up build times
```

Comment:
The idea here was to add a docker image when our resident docker maestro is available again (if I recall correctly)